### PR TITLE
fixs typo: dont -> don't

### PR DESCRIPTION
**Pull request by JoeWrightss**
fixs typo error: dont -> don't
/assign @ryane

### CODE REVIEW

File: install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
       mode: ISTIO_MUTUAL
 ---
 # Destination rule to disable (m)TLS when talking to API server, as API server doesn't have sidecar.
-# Customer should add similar destination rules for other services that dont' have sidecar.
+# Customer should add similar destination rules for other services that don't' have sidecar.
```
Comment (geeknoid) [inline install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml]: don't' -> don't

**Comment by JoeWrightss** [comment]
I signed it!

**Comment by JoeWrightss** [comment]
I signed it!

File: install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml (hunk 1 above)
Comment (JoeWrightss) [reply install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml]: It is fixed, please review again @geeknoid

**Pull request merged**